cou/apps/auxiliary.py

```diff
@@ -54,30 +54,21 @@ def is_valid_track(self, charm_channel: str) -> bool:
             return True
 
         track = self._get_track_from_channel(charm_channel)
-        return (self.charm, self.series, track) in TRACK_TO_OPENSTACK_MAPPING
-
-    @property
-    def possible_current_channels(self) -> list[str]:
-        """Return the possible current channels based on the series and current OpenStack release.
-
-        :return: The possible current channels for the application.
-        :rtype: list[str]
-        :raises ApplicationError: When cannot find tracks.
-        """
         tracks = OPENSTACK_TO_TRACK_MAPPING.get(
             (self.charm, self.series, self.current_os_release.codename)
         )
-        if tracks:
-            return [f"{track}/stable" for track in tracks]
+        return (
+            self.charm,
+            self.series,
+            track,
+        ) in TRACK_TO_OPENSTACK_MAPPING and bool(tracks)
```

Comment:
what is `bool(tacks)` for?

Reply:
It's just to be sure that for a certain charm in a ubuntu series and in a OpenStack release there is at least one possible track. I've changed a little bit and now is checking `len(possible_tracks) > 0
`